tests/test_oidc.py

```diff
@@ -272,3 +280,21 @@ def test_endpoint_with_refresh_configures_correctly():
 
         assert auth.token_url == f"{authority_url}token"
         assert auth.refresh_data["client_id"] == client_id
+
+
+def test_token_logging_outputs_token_to_logs(caplog, monkeypatch):
```

Comment:
Would be good to also check that creating an OIDCSessionFactory with the env var set correctly warns the user that PII will be logged. That would also test that the env var is correctly parsed into the variable here.

Reply:
I think so, it probably makes it easier for users too, if it doesnt work you can change the env var and have it work, rather than having to reopen python